dotcom-rendering/scripts/webpack/webpack.config.browser.js

```diff
@@ -98,19 +98,33 @@ const getLoaders = (bundle) => {
  */
 module.exports = ({ bundle, sessionId }) => ({
 	entry: {
-		sentryLoader: './src/web/browser/sentryLoader/init.ts',
-		bootCmp: './src/web/browser/bootCmp/init.ts',
+		index: './src/web/browser/index.ts',
+		debug: './src/web/browser/debug/index.ts',
 		ga: './src/web/browser/ga/init.ts',
-		ophan: './src/web/browser/ophan/init.ts',
-		islands: './src/web/browser/islands/init.ts',
-		dynamicImport: './src/web/browser/dynamicImport/init.ts',
-		atomIframe: './src/web/browser/atomIframe/init.ts',
-		embedIframe: './src/web/browser/embedIframe/init.ts',
-		newsletterEmbedIframe:
-			'./src/web/browser/newsletterEmbedIframe/init.ts',
-		relativeTime: './src/web/browser/relativeTime/init.ts',
-		initDiscussion: './src/web/browser/initDiscussion/init.ts',
-		debug: './src/web/browser/debug/init.ts',
+	},
+	optimization: {
+		splitChunks: {
+			cacheGroups: {
+				// our own chunk, which is shared between all bundles
+				frameworks: {
+					test: /[\\/]node_modules[\\/](preact|react-is|hoist-non-react-statistics|swr|@emotion|stylis)[\\/]/,
+					chunks: 'all',
+					name: 'frameworks',
+					enforce: true,
+				},
+				// webpack defaults
+				defaultVendors: {
+					test: /[\\/]node_modules[\\/]/,
+					priority: -10,
+					reuseExistingChunk: true,
+				},
+				default: {
+					minChunks: 2,
+					priority: -20,
+					reuseExistingChunk: true,
+				},
```

Comment:
Should we add a bit more information about where to find documentation on this stuff? Feels like a lot of magic numbers.